repos/rust-lang/crates.io-index.toml

```diff
@@ -0,0 +1,13 @@
+org = "rust-lang"
+name = "crates.io-index"
+description = "Registry index for crates.io"
+homepage = "https://crates.io"
+bots = []
+
+[access.teams]
+crates-io = "write"
+crates-io-on-call = "write"
+
+[[branch-protections]]
+pattern = "master"
+required-approvals = 0
```

Comment:
unless I'm missing something, the `pr-required = false` from the PR text seems quite important here, since the crates.io backend is pushing directly to the master branch

Reply:
Indeed. The issue is that `pr-required` is not implemented yet :laughing: